src/components/pieChart/index.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import View from '../view';
+import PieSegment, {PieSegmentProps} from './PieSegment';
+import {SvgPackage} from '../../optionalDependencies';
+const {Svg, Path} = SvgPackage;
+
+export type PieChartSegmentProps = Pick<PieSegmentProps, 'percentage' | 'color'>;
+
+export type PieChartProps = {
+  /**
+   * Pie chart segments array
+   */
+  segments: PieChartSegmentProps[];
+  /**
+   * Pie chart diameter
+   */
+  diameter?: number;
+} & Pick<PieSegmentProps, 'dividerWidth' | 'dividerColor'>;
+
+const DEFAULT_DIAMETER = 144;
+
+const PieChart = (props: PieChartProps) => {
+  const {segments, diameter = DEFAULT_DIAMETER, ...others} = props;
+
+  if (!Svg || !Path) {
+    console.error(`RNUILib PieChart requires installing "@react-native-svg" dependency`);
+  }
```

Comment:
Let's return here null after showing the error to avoid an actual exception that might crash the whole screen or tests